common/changes/office-ui-fabric-react/eddrost-Persona_PersonaSize_addition_2019-08-15-21-03.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
```

Comment:
Delete this file. It's an old format of change logs generated by `rush`. Looks like you had an outdated master when you initially generated this one and now that you updated you have the correct one and this one needs to be gone. In fact the entire common folder is gone now.

Reply:
@mtennoe, do you have any news from Matt? Best case scenario in case if he approves introduction of new sizes is for him to port the specs into the toolkit so we don't end up accidentally deprecating a size in a future major version when trying to align with what we have in the toolkit.

Reply:
I'll talk with him tomorrow to figure out the current state. Will update you as soon as I have that info